src/rust/ide/view/src/documentation.rs

```diff
@@ -141,10 +141,17 @@ impl Model {
         if string.is_empty() {
             Ok(PLACEHOLDER_STR.into())
         } else {
-            let parser    = parser::DocParser::new()?;
-            // FIXME [MM]:  Removes characters that are not supported by Doc Parser yet.
-            //              https://github.com/enso-org/enso/issues/1063
-            let processed = string.replace("\\n", "\n").replace("\"", "");
+            let parser        = parser::DocParser::new()?;
+            let mut processed = string.to_string();
+            let lines = processed.lines().collect::<Vec<&str>>();
+            if lines.len() > 1 {
+                let mut indent = 0;
+                for char in lines[1].chars() {
+                    if char != ' ' { break; }
+                    indent  += 1;
+                }
+                processed = format!{"{}{}",String::from(" ").repeat(indent),processed};
```

Comment:
this code is magical, it has not documentation and there is no explanation what it really does. It is inside a function `gen_html_from`, but it definitely is not generating HTML, so it needs proper documentation / refactoring to other, well named function